frontend/app/src/hocs/withScreencast/components/ScreencastDialog/ScreencastDialog.tsx

```diff
@@ -91,7 +92,10 @@ class ScreencastDialog extends PureComponent<Props, State> {
             </StyledRecordAudioLabel>
           </p>
           <StyledInstruction data-testid="stScreencastInstruction">
-            Press <kbd>Esc</kbd> any time to stop recording.
+            <StreamlitMarkdown
```

Comment:
I decided to use `StreamlitMarkdown` here since `StyledInlineCode` isn't officially exposed from lib. In general, it could be a nice idea to reuse `StreamlitMarkdown` in other similar situations as well where we want to render formatted text.